src/main/native/windows/file.h

```diff
@@ -85,6 +94,13 @@ HANDLE OpenDirectory(const WCHAR* path, bool read_write);
 wstring CreateJunction(const wstring& junction_name,
                        const wstring& junction_target);
 
+// Deletes the file or directory at `path`.
```

Comment:
Nit: Can you add a comment saying that this function will fail if the directory is not empty?

Reply:
Done.